torch/_inductor/codegen/wrapper.py

```diff
@@ -1150,6 +1151,18 @@ def define_user_defined_triton_kernel(self, kernel, configs, kwargs):
         symbols_included = {original_name}
 
         def traverse(cur_kernel):
+            # here we extract the unqualified names (i.e., not attributes and
+            # without prepended module name) loaded in the kernel code, which
+            # are matched with the co_names and __globals__ below to codegen
+            # the respective imports necessary for the kernel compilation
+            unqualified_loads = {
+                node.id
+                for node in ast.walk(ast.parse(cur_kernel.src))
```

Comment:
@jansel does it sound fine to you to use `ast` here? The purpose is extracting unqualified names from the kernel code as candidates for the globals imported externally. Thanks!

Reply:
I think this would be a better way to do the same thing:
```python
>>> {inst.argval for inst in dis.Bytecode(add_kernel_with_imported_symbol)
     if inst.opcode == dis.LOAD_GLOBAL}
{'tl', 'fast_dividef'}
```